CMakeLists.txt

```diff
@@ -523,7 +523,7 @@ if(WIN32)
     # build with multiple cores
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /MP")
-    add_definitions(-D__SSSE3__ -D_CRT_SECURE_NO_WARNINGS)
+    add_definitions(-D__SSSE3__ -D__AVX2__ -D_CRT_SECURE_NO_WARNINGS)
```

Comment:
I'm trying to understand why __AVX2__ definition is necessary. 
If it is not related to Android compilation, I'd prefer to review it in a separate PR.
To the best of my understanding, it is not required.

Reply:
The AVX instructioins are not supported by the compiler in NDK.

The function _mm256_shuffle_epi8 in image_avx.cpp is AVX2 instruction,
according to https://software.intel.com/sites/landingpage/IntrinsicsGuide

And, the code in librealsense/src/proc/pointcloud.cpp arenot AVX2 code. so I keep old one.

Reply:
Yes, but all AVX related stuff should be hidden behind SSE flag, so compiler that doesn't support SSE will not encounter AVX either.

Reply:
I should remove the AVX2 in this line.
the macro \_\_AVX2\_\_ is predefined by MSVC.
https://msdn.microsoft.com/en-us/library/b0084kay.aspx

BTW, the compiler in Android NDK for x86_64 platform have \_\_SSSE3\_\_ macro, but no \_\_AVX2\_\_. so this macro should be kept.
\_\_SSSE3\_\_ and \_\_AVX2\_\_ are predifined macros in gcc/clang